### PR TITLE
Fix issue #3 - <select> style

### PR DESCRIPTION
**Pull request by joaobarbosa**

### CODE REVIEW

**Comment by vkeusebio** [comment]
May I suggest to squash the commits into one?

**Comment by joaobarbosa** [comment]
Is there any way to do this without creating a new pull request, @vkeusebio?

**Comment by fabienwang** [comment]
Hi @joaobarbosa on your select-caret branch: do
git rebase -i HEAD~5
this will merge the last 5 commits you made.

**Comment by joaobarbosa** [comment]
@fabienwang great, I'll do this right now :)

**Comment by fabienwang** [comment]
humm @joaobarbosa you'll have to change "pick" by "squash" in the rebase operation to merge them.
Read it: http://eli.thegreenplace.net/2014/02/19/squashing-github-pull-requests-into-a-single-commit

Regards.

**Comment by joaobarbosa** [comment]
@fabienwang I guess its almost done! hahah after rebase it asked for a merge, should I try it again or leave it like this?

**Comment by vkeusebio** [comment]
Yeah, it should ask for a merge after squashing commits. Looks fine to me @joaobarbosa :ok_hand:

**Comment by joaobarbosa** [comment]
Thanks @vkeusebio!

File: src/_Base.sass [hunk 1]
```diff
@@ -21,4 +21,4 @@ body
 *,
 *:after,
 *:before
-	box-sizing: inherit
+	box-sizing: border-box
```
Comment (cjpatoilo) [inline src/_Base.sass]: Here, it is better to let inherited from the root.

File: dist/milligram.css [hunk 2]
```diff
@@ -175,6 +174,10 @@ select:focus {
   outline: 0;
 }
 
+select {
+  appearance: menulist;
```
Comment (cjpatoilo) [inline dist/milligram.css]: This way each browser renders your way. I plan to do another way. It has to have the same behavior in all browsers.

File: src/_Form.sass [hunk 3]
```diff
@@ -25,6 +25,9 @@ select
 		border: .1rem solid $color-primary
 		outline: 0
 
+select
+	appearance: menulist
```
Comment (cjpatoilo) [inline src/_Form.sass]: This way each browser renders your way. I plan to do another way. It has to have the same behavior in all browsers.

**Comment by cjpatoilo** [comment]
Thank you so much for the PR.
But i will — unfortunately — not merge it because there are some changes directly in CSS files, and not in the source code.

In a way it's my fault. I need to urgently make available a `contributing.md` file with these informations: *"How to contribute for the project"*.

**Comment by joaobarbosa** [comment]
I did changes in the .sass files, and Grunt did the rest of the job. :/

Box-sizing change I was my mistake, I haven't noticed that html was already set to border-box. :+1: 

About the select element, I think it could be set to menulist until another way is defined for it, so, at least, the browser render gets it done for a while. I will try to think about a solution for it. :)

**Pull request closed without merging**